src/versioning/Versioning.ts

```diff
@@ -47,13 +52,14 @@ export default class Versioning {
    */
   static get sdkVersionSemVer(): SemVer {
     const v = VERSION.semverString.match(
-      /^(?<major>[0-9]+)\.(?<minor>[0-9]+)(?:\.(?<patch>[0-9]+))?/
+      /^(?<major>[0-9]+)\.(?<minor>[0-9]+)((?:\.(?<patch>[0-9]+))(?:-(?<pre_release>[a-zA-Z]+(\.[0-9])*))?)?/
     );
 
     return {
       major: v?.groups?.major,
       minor: v?.groups?.minor,
       patch: v?.groups?.patch,
+      pre_release: v?.groups?.pre_release,
```

Comment:
Just curious, why we use snake case here? why not `preRelease`?

Reply:
oops. Just me with ruby style naming :)). Fixed.